docs/configuration.rst

```diff
@@ -0,0 +1,35 @@
+-------------
+Configuration
+-------------
+
+Each part of gardener project (smart contracts repo, server, monitor) contains .env variables file.
+This section is going to explain parameters from them. Default parameters are set in a way everything should work correctly when using local test blockchain (ganache) and following getting-started guide.
```

Comment:
```suggestion
This section explains the available configuration parameters. Default parameters are set so that everything should work when using local test blockchain (ganache) and following the getting-started guide.
```
Link the getting started guide?